src/runtime/pay-flow.js

```diff
@@ -126,7 +126,15 @@ export class PayStartFlow {
     }
 
     // Start/cancel events.
-    this.deps_.callbacks().triggerFlowStarted(SubscriptionFlows.SUBSCRIBE, req);
+    if (this.productType_ == ProductType.SUBSCRIPTION) {
```

Comment:
Revert this change and keep it as a separate PR (to fix the trigger).

Reply:
Huh, I thought I'd reverted this change on this branch. Thanks for catching that!